operator/gitops/argocd/argo-apps/tekton-chains.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     server: https://kubernetes.default.svc
   source:
     path: operator/gitops/argocd/tekton-chains/overlays/openshift
-    repoURL: https://github.com/openshift-pipelines/pipeline-service.git
+    repoURL: https://github.com/AndrienkoAleksandr/pipelines-service.git
```

Comment:
Any reason why you need to test with your fork of pipelines?

Reply:
No, that's only for case if in the original repo was changed folder location due refactoring or someting like that.